tests/Transforms/tosa_to_boolean_tfhe/hello_world_clean.mlir

```diff
@@ -5,11 +5,11 @@ module attributes {tf_saved_model.semantics} {
 
   func.func @main(%arg0: tensor<1x1xi8> {iree.identifier = "serving_default_dense_input:0", tf_saved_model.index_path = ["dense_input"]}) -> (tensor<1x1xi32> {iree.identifier = "StatefulPartitionedCall:0", tf_saved_model.index_path = ["dense_2"]}) attributes {tf_saved_model.exported_names = ["serving_default"]} {
     %0 = "tosa.const"() {value = dense<429> : tensor<1xi32>} : () -> tensor<1xi32>
-    %1 = "tosa.const"() {value = dense<[[-39, 59, 39, 21, 28, -32, -34, -35, 15, 27, -59, -41, 18, -35, -7, 127]]> : tensor<1x16xi8>} : () -> tensor<1x16xi8>
-    %2 = "tosa.const"() {value = dense<[-729, 1954, 610, 0, 241, -471, -35, -867, 571, 581, 4260, 3943, 591, 0, -889, -5103]> : tensor<16xi32>} : () -> tensor<16xi32>
+    %1 = "tosa.const"() {value = dense<[[39, 59, 39, 21, 28, 32, 34, 35, 15, 27, 59, 41, 18, 35, 7, 127]]> : tensor<1x16xi8>} : () -> tensor<1x16xi8>
```

Comment:
qq: was this done manually to support testing with the unsigned int restriction?